pkg/agent/flowexporter/connections/connections_test.go

```diff
@@ -255,13 +285,45 @@ func TestConnectionStore_addAndUpdateConn(t *testing.T) {
 
 			ingressOfID := binary.LittleEndian.Uint32(test.flow.Labels[:4])
 			npQuerier.EXPECT().GetNetworkPolicyByRuleFlowID(ingressOfID).Return(&np1)
+			npQuerier.EXPECT().GetRuleByFlowID(ingressOfID).Return(&rule1)
 			expConn.IngressNetworkPolicyName = np1.Name
 			expConn.IngressNetworkPolicyNamespace = np1.Namespace
+			expConn.IngressNetworkPolicyUID = string(np1.UID)
+			switch np1.Type {
+			case cpv1beta.K8sNetworkPolicy:
+				expConn.IngressNetworkPolicyType = 1
+			case cpv1beta.AntreaNetworkPolicy:
+				expConn.IngressNetworkPolicyType = 2
+			case cpv1beta.AntreaClusterNetworkPolicy:
+				expConn.IngressNetworkPolicyType = 3
+			}
+			expConn.IngressNetworkPolicyRuleName = rule1.Name
+			if expConn.IngressNetworkPolicyType == 1 {
+				expConn.IngressNetworkPolicyRulePriority = -1
+			} else {
+				expConn.IngressNetworkPolicyRulePriority = int32(*rule1.Priority)
+			}
 
 			egressOfID := binary.LittleEndian.Uint32(test.flow.Labels[4:8])
 			npQuerier.EXPECT().GetNetworkPolicyByRuleFlowID(egressOfID).Return(&np2)
+			npQuerier.EXPECT().GetRuleByFlowID(egressOfID).Return(&rule2)
 			expConn.EgressNetworkPolicyName = np2.Name
 			expConn.EgressNetworkPolicyNamespace = np2.Namespace
+			expConn.EgressNetworkPolicyUID = string(np2.UID)
+			switch np2.Type {
+			case cpv1beta.K8sNetworkPolicy:
+				expConn.EgressNetworkPolicyType = 1
```

Comment:
we can use const defined in go-ipfix instead of self-assigned value, same for next several uses.